core/types/transaction.go

```diff
@@ -332,13 +336,13 @@ func (tx *Transaction) String() string {
 	From:     %s
 	To:       %s
 	Nonce:    %v
-	GasPrice: %v
-	GasLimit  %v
-	Value:    %v
+	GasPrice: %#x
+	GasLimit  %#x
+	Value:    %#x
```

Comment:
There will now display the `GasPrice`, `GasLimit` and `Value` in hex, not decimal. Not sure if that's a problem or not?